Automation/setup/ambarinode_setup.sh

```diff
@@ -83,12 +83,37 @@ function patch_ambari {
 }
 
 function blueprint_deploy {
-    #REST connection in deploy_from_blueprint.py failing for no reason...
+    #REST connection in deploy_from_blueprint.py can fail, so keep trying till success is reached
     local command="$BIN_DIR/blueprint_deploy.sh $VERSION ${KAVE_BLUEPRINT%.*} ${KAVE_CLUSTER%.*} $WORKING_DIR"
-    until $command; do
-	echo "Blueprint installation failed, retrying..."
-	sleep 10
+    # try for a while(), then backup plan
+    local count=5
+    local blueprint_trials=5 
```

Comment:
Not sure about this - would't it reset every time this function is called recurrently (cf. line 107) ? So infinite loop indeed :)